api/v1alpha1/nutanix_clusterconfig_types.go

```diff
@@ -76,3 +84,39 @@ func (s NutanixPrismCentralEndpointSpec) ParseURL() (string, int32, error) {
 
 	return hostname, int32(port), nil
 }
+
+// NutanixFailureDomains is a list of FDs.
+type NutanixFailureDomains []NutanixFailureDomain
+
+// NutanixFailureDomain configures failure domain information for Nutanix.
+type NutanixFailureDomain struct {
```

Comment:
Do we expect clients to set the FD name in MD to use these, maybe some docs on how to use this would be good?

Reply:
yes we have a internal tracker for opendocs for adding capx docs. but we can also update caren docs.

Reply:
docs added.